src/components/Widget/Widget.tsx

```diff
@@ -0,0 +1,36 @@
+import {WidgetProps} from "./types";
+import React from "react";
+import { HintStyled, WidgetStyled, WidgetInfoBox, WidgetTitle, WidgetInfo } from "./Widget.style";
+import colors from "../../styles/colors";
+import {Icon} from "../Icon";
+import {Tooltip} from "../Tooltip"
+import {iconTypes} from "../Icon/collection";
+
+const getHint = (hint: string) => {
+    return (
+        <HintStyled>
+            <Tooltip text={hint} children={[<Icon fill={colors.blue} size={14} svg={iconTypes.helpCircle}/>]} />
+        </HintStyled>
+    )
+}
+
+const Widget: React.FC<WidgetProps> = ({
+    id,
+    description,
+    hint,
+    icon,
+    title
+}: WidgetProps) => {
+    return (
+        <WidgetStyled id={id}>
+            {icon && <div style={{display: "grid", placeItems: "center", marginRight: '10px'}}><Icon size={30} fill={colors.blue} svg={icon}/></div>}
```

Comment:
ok now i see the gird, excuse one of previous comments, please move this to styles file =]